samples/MortgageApplication/application-conf/application.properties

```diff
@@ -45,18 +45,18 @@ skipImpactCalculationList=
 
 # ### Properties to enable and configure build property overrides using individual artifact properties files
 
-# flag to enable the zAppBuild capability to load individual artifact properties files for individual source files, 
-# Note: To activate the loading of property files for a group of files, it is recommended to use the file property path
-# syntax instead. (i.e. loadFileLevelProperties=true :: **/cobol/*.cbl to enable it for all cobol files)
+# flag to enable the zAppBuild capability to load individual artifact properties files for all individual source files. 
```

Comment:
Just double-checking if specifying "all" individual source files here (in line 48) is an accurate clarification.